components/cli/internal/test/kubecli.go

```diff
@@ -53,46 +61,56 @@ func NewMockKubeCli(opts ...option) *MockKubeCli {
 }
 
 // GetCells returns cell instances array.
-func (kubecli *MockKubeCli) GetCells() ([]kubernetes.Cell, error) {
-	return kubecli.cells.Items, nil
+func (kubeCli *MockKubeCli) GetCells() ([]kubernetes.Cell, error) {
+	return kubeCli.cells.Items, nil
 }
 
-func (kubecli *MockKubeCli) GetComposites() ([]kubernetes.Composite, error) {
-	return kubecli.composites.Items, nil
+func (kubeCli *MockKubeCli) GetComposites() ([]kubernetes.Composite, error) {
+	return kubeCli.composites.Items, nil
 }
 
-func (kubecli *MockKubeCli) GetCell(cellName string) (kubernetes.Cell, error) {
-	for _, cell := range kubecli.cells.Items {
+func (kubeCli *MockKubeCli) GetCell(cellName string) (kubernetes.Cell, error) {
+	for _, cell := range kubeCli.cells.Items {
 		if cell.CellMetaData.Name == cellName {
 			return cell, nil
 		}
 	}
 	return kubernetes.Cell{}, fmt.Errorf("cell %s does not exist", cellName)
 }
 
-func (kubecli *MockKubeCli) GetComposite(compositeName string) (kubernetes.Composite, error) {
-	for _, composite := range kubecli.composites.Items {
+func (kubeCli *MockKubeCli) GetComposite(compositeName string) (kubernetes.Composite, error) {
+	for _, composite := range kubeCli.composites.Items {
 		if composite.CompositeMetaData.Name == compositeName {
 			return composite, nil
 		}
 	}
 	return kubernetes.Composite{}, fmt.Errorf("composite %s does not exist", compositeName)
 }
 
-func (kubecli *MockKubeCli) DeleteResource(kind, instance string) (string, error) {
+func (kubeCli *MockKubeCli) DeleteResource(kind, instance string) (string, error) {
 	return "", nil
 }
 
-func (kubecli *MockKubeCli) SetVerboseMode(enable bool) {
-}
-
-func (kubecli *MockKubeCli) GetInstancesNames() ([]string, error) {
+func (kubeCli *MockKubeCli) GetInstancesNames() ([]string, error) {
 	var instanceNames []string
-	for _, cell := range kubecli.cells.Items {
+	for _, cell := range kubeCli.cells.Items {
 		instanceNames = append(instanceNames, cell.CellMetaData.Name)
 	}
-	for _, composites := range kubecli.composites.Items {
+	for _, composites := range kubeCli.composites.Items {
 		instanceNames = append(instanceNames, composites.CompositeMetaData.Name)
 	}
 	return instanceNames, nil
 }
+
+func (kubeCli *MockKubeCli) GetInstanceBytes(instanceKind, InstanceName string) ([]byte, error) {
+	if instanceKind == "cells.mesh.cellery.io" {
+		return kubeCli.cellsBytes[InstanceName], nil
+	} else if instanceKind == "composites.mesh.cellery.io" {
+		return kubeCli.cellsBytes[InstanceName], nil
```

Comment:
extract constants for cells.mesh.cellery.io and composites.mesh.cellery.io, or re-use if there is an existing constant.

Reply:
Updated.